src/base-input.js

```diff
@@ -6,6 +6,7 @@
 /* eslint-disable max-lines */
 /* eslint no-magic-numbers: ["error", { "ignore": [0] }] */
 /* eslint-disable max-statements */
+/* eslint-disable prefer-regex-literals */
```

Comment:
With the regex removed I don't think we need this.

Reply:
I will remove that.